### PR TITLE
change address prefix and add readme doc

### PR DESCRIPTION
**Pull request by guoshijiang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+## Contributing
+
+Looking for a good place to start contributing? Check out some [`good first issues`](https://github.com/bitdao-io/bitos/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+For additional instructions, standards and style guides, please refer to the [Contributing](./CONTRIBUTING.md) document.
+
+## Careers
+
+See our open positions on [Cosmos Jobs](https://jobs.cosmos.network/project/bitos-d0sk1uxuh-remote/), [Notion](https://tharsis.notion.site), or feel free to [reach out](mailto:careers@thars.is) via email.
```
Comment (silmaj) [inline README.md]: cosmos job？

**Review by wukongcheng — APPROVED** [review]
LGTM

**Review by silmaj — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (guoshijiang) [reply README.md]: already fix

File: README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  order: false
+-->
+
+<div align="center">
+  <h1> Bitos </h1>
+</div>
+
+<div align="center">
+  <a href="#">
+    <img alt="Version" src="https://img.shields.io/github/tag/tharsis/bitos.svg" />
```
Comment (chengwenxi) [inline README.md:12]:
```suggestion
    <img alt="Version" src="https://img.shields.io/github/tag/bitdao-io/bitos.svg" />
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  </a>
+  <a href="#">
+    <img alt="License: Apache-2.0" src="https://img.shields.io/github/license/tharsis/bitos.svg" />
```
Comment (chengwenxi) [inline README.md:15]:
```suggestion
    <img alt="License: Apache-2.0" src="https://img.shields.io/github/license/bitdao-io/bitos.svg" />
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  </a>
+  <a href="#">
+    <img alt="GoDoc" src="https://godoc.org/github.com/bitdao-io/bitos?status.svg" />
+  </a>
+  <a href="#">
+    <img alt="Go report card" src="https://goreportcard.com/badge/github.com/bitdao-io/bitos"/>
+  </a>
+  <a href="#">
+    <img alt="Lines of code" src="https://img.shields.io/tokei/lines/github/tharsis/bitos">
```
Comment (chengwenxi) [inline README.md:24]:
```suggestion
    <img alt="Lines of code" src="https://img.shields.io/tokei/lines/github/bitdao-io/bitos">
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+</div>
+<div align="center">
+  <a href="#">
+    <img alt="Discord" src="https://img.shields.io/discord/809048090249134080.svg" />
+  </a>
+  <a href="#">
+    <img alt="Lint Status" src="https://github.com/bitdao-io/bitos/actions/workflows/lint.yml/badge.svg?branch=main" />
+  </a>
+  <a href="#">
+    <img alt="Code Coverage" src="https://codecov.io/gh/tharsis/bitos/branch/main/graph/badge.svg" />
```
Comment (chengwenxi) [inline README.md:35]:
```suggestion
    <img alt="Code Coverage" src="https://codecov.io/gh/bitdao-io/bitos/branch/main/graph/badge.svg" />
```

File: ibc/utils_test.go [hunk 6]
```diff
@@ -55,7 +55,7 @@ func TestGetTransferSenderRecipient(t *testing.T) {
 				Data: transfertypes.ModuleCdc.MustMarshalJSON(
 					&transfertypes.FungibleTokenPacketData{
 						Sender:   "cosmos1",
-						Receiver: "bitos1x2w87cvt5mqjncav4lxy8yfreynn273xn5335v",
+						Receiver: "bit1x2w87cvt5mqjncav4lxy8yfreynn273xn5335v",
```
Comment (chengwenxi) [inline ibc/utils_test.go:58]: The address suffix is the checksum value, I don't sure the test will work if we don't update it.

**Review by silmaj — APPROVED** [review]

File: ibc/utils_test.go (hunk 6 above)
Comment (guoshijiang) [reply ibc/utils_test.go:58]: I will fix it

**Review by chengwenxi — APPROVED** [review]
lgtm

**Pull request merged**